bundles/org.openhab.binding.netatmo/README.md

```diff
@@ -649,18 +649,19 @@ Person things are automatically created in discovery process for all known perso
 
 **Supported channels for the Person thing:**
 
-| Channel Group | Channel ID   | Item Type | Description                                            |
-| ------------- | ------------ | --------- | ------------------------------------------------------ |
-| person        | avatar-url   | String    | URL for the avatar of this person                      |
-| person        | avatar       | Image     | Avatar of this person                                  |
-| person        | at-home      | Switch    | Indicates if this person is known to be at home or not |
-| person        | last-seen    | DateTime  | Moment when this person was last seen                  |
-| last-event    | subtype      | String    | Sub-type of event                                      |
-| last-event    | message      | String    | Last event message from this person                    |
-| last-event    | time         | DateTime  | Moment of the last event for this person               |
-| last-event    | snapshot     | Image     | Picture of the last event for this person              |
-| last-event    | snapshot-url | String    | URL for the picture of the last event for this person  |
-| last-event    | camera-id    | String    | ID of the camera that triggered the event              |
+| Channel Group  | Channel ID   | Item Type | Description                                                              |
+| -------------- | ------------ | --------- | ------------------------------------------------------------------------ |
+| security-event | home-event   |           | Trigger channel which is triggered an event for this person is triggered |
```

Comment:
_Trigger channel which is triggered an event for this person is triggered_

That might need some improvement.

Reply:
An event for this person has been triggered ?

Reply:
On what occasions? When the person enters or leaves?

Reply:
When the person has been identified by a camera.

Reply:
Suggestion:
_Trigger channel which is triggered by an event for this person (PERSON, PERSON_AWAY, PERSON_HOME)_